junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/extension/TestInfoParameterResolver.java

```diff
@@ -85,7 +85,7 @@ public String toString() {
 		}
 
 		private static Object nullSafeGet(Optional<?> optional) {
-			return optional != null ? optional.orElse(null) : null;
+			return optional.isPresent() ? optional.orElse(null) : null;
```

Comment:
This change breaks null-safety. `null.isPresent()` will crash and burn.

Reply:
Thanks for the review, you are right, I didn't think about that situation